scripts/files/RtfThemeLarge.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information. -->
+<!-- Downloaded from https://github.com/wixtoolset/wix/blob/v5.0.2/src/ext/Bal/stdbas/Resources/RtfLargeTheme.xml -->
```

Comment:
We likely need to include the license text. We can probably do it as a comment in this file to avoid writing another `LICENSE.xyz.TXT` file in the repo.